frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -122,6 +122,13 @@ interface JwtToken {
 }
 
 const userId = (): string => {
+  if (process.env.NODE_ENV === "development") {
+    // eslint-disable-next-line
+    console.warn("No user token set in development");
```

Comment:
Should this be after the if check? I'm confused what the warning is meant to entail.

Reply:
Oh it's basically to denote to users that dev will always be anonymous - I can also remove it

Reply:
Oh ok, yeah I'd say after the if check if we're not returning something it might be nice to say 'Returning `Anonymous` user for development'. I was confused at first when I had to figure out why it was getting that.